apps/framework-cli/src/framework/controller.rs

```diff
@@ -137,23 +140,17 @@ pub(crate) async fn create_or_replace_view(
     Ok(())
 }
 
-pub(crate) async fn create_or_replace_table(
+pub(crate) async fn create_or_replace_tables(
     fo: &FrameworkObject,
     configured_client: &ConfiguredDBClient,
-) -> Result<(), Error> {
+) -> anyhow::Result<()> {
     info!("Creating table: {:?}", fo.table.name);
-    let create_table_query = fo.table.create_table_query().map_err(|e| {
-        Error::new(
-            ErrorKind::Other,
-            format!("Failed to get clickhouse query: {:?}", e),
-        )
-    })?;
-    let drop_table_query = fo.table.drop_table_query().map_err(|e| {
```

Comment:
The `map_err` calls add quite a lot of noise. We don't need to convert the error by using anyhow. If we do want to add extra context later, we can rethink a better abstraction.